examples/analytics.amp.html

```diff
@@ -22,9 +22,9 @@
 <amp-analytics id="analytics1">
 <script type="application/json">
 {
-  "host": "example.com",
+  "transport": {"beacon": false, "xhrpost": false},
```

Comment:
In vendors.js, the "default" config (which all configs inherit from) has `beacon`, `xhrpost`, and `image` all set to `true`. This means the default behavior will be beacon > xhrpost > image unless one of those options are specifically disabled in the vendor, inline, or remote config.